test/index.test.ts

```diff
@@ -1,8 +1,85 @@
 import { expect, it, describe } from "vitest";
-import {} from "../src";
+import {
+  bold,
+  blockquote,
+  boldAndItalic,
+  codeBlock,
+  heading,
+  hr,
+  image,
+  italic,
+  link,
+  list,
+  strikethrough,
+  table
+} from "../src";
 
 describe("omark", () => {
-  it.todo("pass", () => {
-    expect(true).toBe(true);
+  it("bold", () => {
+    expect(bold("Hello, World!")).toBe("**Hello, World!**");
   });
+
+  it("blockquote", () => {
+    expect(blockquote("Hello, World!")).toBe("> Hello, World!");
+  });
+
+  it("boldAndItalic", () => {
+    expect(boldAndItalic("Hello, World!")).toBe("***Hello, World!***");
+  });
+
+  it("coldBlock", () => {
+    expect(codeBlock('console.log("Hello, World!");', "js")).toBe(
+      '```js\nconsole.log("Hello, World!");\n```',
+    );
+  });
+
+  it("heading", () => {
+    expect(heading("Hello, World!", 1)).toBe("\n# Hello, World!\n");
+    expect(heading("Hello, World!", 2)).toBe("\n## Hello, World!\n");
+    expect(heading("Hello, World!", 3)).toBe("\n### Hello, World!\n");
+    expect(heading("Hello, World!", 4)).toBe("\n#### Hello, World!\n");
+    expect(heading("Hello, World!", 5)).toBe("\n##### Hello, World!\n");
+    expect(heading("Hello, World!", 6)).toBe("\n###### Hello, World!\n");
+  });
+
+  it("hr", () => {
+    expect(hr()).toBe("---");
+  });
+
+  it("image", () => {
+    expect(image("https://cataas.com/cat", "Cute Cat")).toBe(
+      "![Cute Cat](https://cataas.com/cat)",
+    );
+  });
+
+  it("italic", () => {
+    expect(italic("Hello, World!")).toBe("_Hello, World!_");
+  });
+
+  it("link", () => {
+    expect(link("Google", "https://www.google.com")).toBe("[https://www.google.com](Google)");
+    expect(link("https://www.google.com", "Google", { external: true, title: "Google" })).toBe(
+      '<a href="https://www.google.com" title="Google" target="_blank">Google</a>',
+    );
+  });
+
+  it("list", () => {
+    expect(list(['Item 1', 'Item 2', 'Item 3'])).toBe("- Item 1\n- Item 2\n- Item 3")
+  })
+
+  it("strikethrough", () => {
+    expect(strikethrough("Hello, World!")).toBe("~~Hello, World!~~")
```

Comment:
Since most of the parts are shared between tests, I am thinking to use a generative system:

```
const tests = [
  {fn: 'strikethrough', input: 'Hello, World!', output: '...'
 ]
```

And then you can use a for loop to make a test for each. 

Let me know if wanted more info how to do it.

Reply:
I appreciate it if you could give me more information on how to.

Reply:
I can do it like below, 

```js
const tests = [
  {fn: 'bold', input: 'Hello, World!', output: '**Hello, World!**'},
]

for (const test of tests) {
  it(`${test.fn}`, () => {
    expect(md[test.fn](test.input)).toBe(test.output)
  })
}
```

but it can look a bit hacky if I want to keep "table" test case as it is.

Reply:
We can keep table seperated 👍🏼

Reply:
done.